### DISCUSSION TITLE
Allow configuring dgs.graphql.path to be '/'

### DISCUSSION THREAD

**Discussion by setchy** [Netflix/dgs-framework#615] [Ideas]
Curious to understand if the current [path validation logic](https://github.com/Netflix/dgs-framework/blob/39acab7a835111729dcfed2a837905cefeb2ee25/graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/DgsWebMvcConfigurationProperties.kt#L59) for `dgs.graphql.path` could be relaxed to allow exposing the API endpoint at the root (ie: http://localhost:8080)

At the moment, this is not allowed behavior.  Would there be any downsides for this specific property only?

**Comment by setchy** [comment]
Some further context - the reason I ask this is so I can decide on a direction to close out adding dgs to the new(ish) [apollo-federation-subgraph-compatibility project](https://github.com/apollographql/apollo-federation-subgraph-compatibility/pull/33)

**Comment by paulbakker** [comment]
PR ready: https://github.com/Netflix/dgs-framework/pull/622

**Reply by setchy** [reply]
Thanks @paulbakker 👏

**Reply by berngp** [reply]
Available in 4.7.6
